akka-http-core/src/test/scala/akka/http/scaladsl/model/UriSpec.scala

```diff
@@ -721,11 +722,23 @@ class UriSpec extends WordSpec with Matchers {
       Uri("https://host:3030/").withPort(4450).effectivePort shouldEqual 4450
     }
 
```

Comment:
We don't have a test for java URI <=> Akka Uri roundtrip, If we want to support that roundtrip the test should be added and perhaps fuzzy tested with scalacheck?